server/routes/request.ts

```diff
@@ -617,34 +629,38 @@ requestRoutes.put<{ requestId: string }>(
   }
 );
 
-requestRoutes.delete('/:requestId', async (req, res, next) => {
-  const requestRepository = getRepository(MediaRequest);
-
-  try {
-    const request = await requestRepository.findOneOrFail({
-      where: { id: Number(req.params.requestId) },
-      relations: ['requestedBy', 'modifiedBy'],
-    });
+requestRoutes.delete(
+  '/:requestId',
+  isAuthenticated(),
```

Comment:
All request routes require `isAuthenticated()` by default so this is unnecessary.